lib/stopovers.js

```diff
@@ -0,0 +1,114 @@
+'use strict'
+
+const got = require('got')
+const moment = require('moment-timezone')
+const memoize = require('p-memoize')
+const isString = require('lodash.isstring')
+const isDate = require('lodash.isdate')
+const groupBy = require('lodash.groupby')
+const merge = require('lodash.merge')
+
+const _stations = require('./stations')
+const stations = memoize(_stations, {maxAge: 24*60*60*1000})
+stations()
+
+const defaults = () => ({
+    when: new Date()
+})
+
+const getKeyFromBoth = (obj1, obj2, key) => {
```

Comment:
Does this function assert that `obj1[key] === `obj2[key]` and return `obj2[key] || obj1[key]`?